Jenkinsfile

```diff
@@ -53,11 +52,29 @@ pipeline{
       }
       agent {label 'devExpLinuxPool'}
       steps{
-        runtests('Latest','11','JAVA11_HOME_DIR')
+        sh label:'mlsetup', script: '''#!/bin/bash
+            echo "Removing any running marklogic server and clean up marklogic data directory"
```

Comment:
Small nit - always do "MarkLogic" in a comment/printout, not "marklogic".

Reply:
Done